docs/source/tutorials/outliers.ipynb

```diff
@@ -2,46 +2,75 @@
  "cells": [
   {
    "cell_type": "markdown",
-   "id": "5decad3c",
+   "id": "1043b220",
    "metadata": {},
    "source": [
-    "# Detecting Outliers with Cleanlab and PyTorch Image Models (timm)\n",
+    "# Detecting Outlier Examples with Cleanlab and PyTorch Image Models (timm)\n",
     "\n",
-    "This 5-minute tutorial shows how to detect outliers in image data using Cleanlab and PyTorch. We use the **cifar10** dataset where each image belongs to 1 of 10 classes: `[airplane, automobile, bird, cat, deer, dog, frog, horse, ship, truck]`. You can easily replace the image dataset + neural network used here with any other Pytorch dataset + neural network (e.g. to instead detect outliers in text data). \n",
+    "This 5-minute tutorial shows how to detect outliers examples in image data using Cleanlab and PyTorch. We use the **cifar10** dataset where each image belongs to 1 of 10 classes: `[airplane, automobile, bird, cat, deer, dog, frog, horse, ship, truck]`. You can easily replace the image dataset + neural network used here with any other Pytorch dataset + neural network (e.g. to instead detect outliers in text data). \n",
     "\n",
     "**Overview of what we'll do in this tutorial:**\n",
     "\n",
+    "Detecting outliers using `feature_embeddings`\n",
     "- Pre-process [cifar10](https://www.cs.toronto.edu/~kriz/cifar.html) to create a Pytorch dataset.\n",
     "- Create `train_data` and `test_data` such that `train_data` only contains images of animals and `test_data` contains images from all classes.\n",
     "- Load pretrained `timm` neural network model and extract `train_data` and `test_data` feature embeddings from its representations.\n",
     "- Use `cleanlab` to find naturally occurring outlier examples in the `train_data` (i.e. atypical images).\n",
     "- Find outlier examples in the `test_data` that do not stem from training data distribution (including out-of-distribution non-animal images).\n",
-    "- Explore threshold selection for determining which images are outliers vs not."
+    "- Explore threshold selection for determining which images are outliers vs not.\n",
+    "\n",
+    "Detecting outliers using `pred_probs`\n",
+    "- Train Logistic Regression classifier on the feature embeddings generated by `timm` model.\n",
```

Comment:
Turn our `timm` model into a classifier by adding an output layer fit via Logistic Regression to the (in-distribution) training data.